docs/internals/continuous_integration.rst

```diff
@@ -1,42 +1,80 @@
-=================================================
-Continuous Integration with Jenkins and Travis CI
-=================================================
+======================
+Continuous Integration
+======================
 
 
 Overview
 ========
 
-The OpenHatch code has a suite of tests. It's important that when we deploy the
-code to the website, all tests pass.
+The OpenHatch code has a suite of tests. It's important that when we deploy
+the code changes to the website that all tests are passing.
 
-Jenkins is a "continuous integration" tool (read `more on Wikipedia`_). It wakes
-up once an hour, checks the git repository for new commits, and runs the test
-suite. For additional information about Jenkins, read `more on Jenkins`_.
+`Continuous integration`_ helps our developers see if their code changes are
+passing all tests or are failing a test and additional code changes are
+needed.
+
+.. _Continuous integration: http://www.aosabook.org/en/integration.html
 
-.. _more on Wikipedia: https://en.wikipedia.org/wiki/Continuous_integration
-.. _more on Jenkins: https://jenkins-ci.org
 
-Travis CI is a hosted, distributed "continuous integration" system (read 
-`more on Wikipedia about Travis CI`_). For additional information about 
-Travis CI, read `more on Travis CI`_.
+Travis CI
+=========
+
+`Travis CI`_ is a hosted, distributed "continuous integration" system (read
+`more on Wikipedia about Travis CI`_). The GitHub page for the `oh-mainline`
```

Comment:
Two remarks here:

* "for the `oh-mainline`" seems like it's missing a noun. "for the `oh-mainline` repo" maybe?

* Also you can link people to https://travis-ci.org/openhatch/oh-mainline if you want. I don't know if it's worth bothering.